core/jazz_ui/e2e/04-jazzservices/jazzservices.e2e-spec.ts

```diff
@@ -217,6 +257,26 @@ describe('Overview', () => {
             browser.wait(EC.visibilityOf(jazzServices_po.getAssetHeader()), timeOutHigh);
             browser.sleep(4000);
             browser.wait(EC.elementToBeClickable(jazzServices_po.getServiceFromAsset()), timeOutHigh);
+      });
+
+      it('Verify Logs' ,  () => {
+            browser.wait(EC.visibilityOf(jazzServices_po.getLogs()), timeOutHigh);
+            jazzServices_po.getLogs().click();
+            browser.wait(EC.visibilityOf(jazzServices_po.getFilterIcon()), timeOutHigh);
+            browser.driver.switchTo().activeElement();
+            jazzServices_po.getFilterIcon().click();
+            jazzServices_po.getDropDown().click();
+            jazzServices_po.getDay().click();
+            browser.sleep(4000);
```

Comment:
do you really need 4 seconds of sleep here and below

Reply:
We can reduce that,for seeing the Logs I just kept 4 seconds.